Resources/config/routing/admin_activation.yaml

```diff
@@ -0,0 +1,11 @@
+fsi_admin_activation:
+    path: /{translatableLocale}/activation/activate/{token}
```

Comment:
Why is the translatable locale needed in activation and change password routes? Is it reasonable for user to have different logins and/or passwords in different locales? The locale that is needed here is the used to translate UI but not entities' properties. But that's a different story.